Firmware/ultralcd.cpp

```diff
@@ -7386,9 +7393,6 @@ static bool lcd_selfcheck_pulleys(int axis)
 			((READ(Y_MIN_PIN) ^ Y_MIN_ENDSTOP_INVERTING) == 1)) {
 			endstop_triggered = true;
 			if (current_position_init - 1 <= current_position[axis] && current_position_init + 1 >= current_position[axis]) {
-				current_position[axis] += (axis == X_AXIS) ? 13 : 9;
```

Comment:
what was the original intention here?

Reply:
Those were the hard coded values for mk2 after it checked for loose pulleys. The selftest does as follows:
MK2/MK2.5(S) : Home X -> Check pulleys and move X axis to Z probe position -> The same thing for Y axis -> Raise Z -> lower Z to point
Newer printers : Home X -> Check Axis length X-> Check Axis length Y -> move to point position -> Raise Z -> lower Z to point
With this change the head is moved to the point while being raised AFTER checking X and Y.